boot/zephyr/boards/nrf54lm20apdk_nrf54lm20a_cpuapp.conf

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+CONFIG_BOOT_MAX_IMG_SECTORS=256
+
+# Ensure that the SPI NOR driver is disabled by default
+CONFIG_SPI_NOR=n
+
+# TODO: below are not yet supported and need fixing
+CONFIG_FPROTECT=n
+
+CONFIG_BOOT_WATCHDOG_FEED=n
+
+# Ensure the fastest RRAM write operations
+CONFIG_NRF_RRAM_WRITE_BUFFER_SIZE=32
```

Comment:
Is this still needed? Isn't that default at this point when radio sync is disabled?